README.md

```diff
@@ -1,52 +1,102 @@
-PHP module for GEOS
-===================
+# PHP module for GEOS
 
-[![build status]
-(https://drone.osgeo.org/api/badges/geos/php-geos/status.svg?branch=master)]
-(https://drone.osgeo.org/geos/php-geos?branch=master)
+[![build status](https://dronie.osgeo.org/api/badges/geos/php-geos/status.svg?branch=master)](https://github.com/libgeos/php-geos/tree/master)
 
 The code in this directory provides a PHP module to make use
-of functionalities of the [GEOS library] (http://geos.osgeo.org).
+of functionalities of the [GEOS library](http://geos.osgeo.org).
 
 The bindings are linked to the C-API, for betters stability.
 
-# Building
+## Building (UNIX)
 
-## Requirements
+### Requirements
 
 You need:
 
-  - PHP development files
-    ( often found in packages named something like 'php5-dev' )
+- PHP development files
+  ( often found in packages named something like 'php5-dev' )
 
-  - GEOS development files
-    ( often found in packages named something like 'libgeos-dev' )
+- GEOS development files
+  ( often found in packages named something like 'libgeos-dev' )
 
-## Procedure
+### Procedure
 
-    git clone https://git.osgeo.org/gitea/geos/php-geos.git
+    git clone https://github.com/libgeos/php-geos.git
```

Comment:
I don't remember a change in official tree publication

Reply:
This is from another user from the PHP 74 branch arrived. Need to return the link back?

Reply:
yes, official url is the osgeo.org one